CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
-cmake_minimum_required(VERSION 3.5)
-
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Given the use of `target_link_directories` below, I feel like we should bump this to `3.13` instead.

Reply:
Indeed, it appears that is [new in 3.13](https://cmake.org/cmake/help/v3.13/command/target_link_directories.html). Implemented in f8cdca6dadc.